mvg/utils/response_processing.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module contains helper functions for manipulating the response from endpoints.
+"""
+
+from enum import Enum
+from typing import Dict, Union, Callable
+
+
+class SortOrder(Enum):
+    ASC = "asc"
+    DESC = "desc"
+
+
+def get_paginated_items(
+    request: Callable, url: str, params: Dict[str, Union[int, SortOrder]]
+) -> Dict:
+    """
+    Retrieves items and total as a dictionary from a paginated endpoint.
+    If offset and limit params are not specified, all items will be returned.
+
+    Parameters
+    ----------
+    request : Callable
+        The request function to be used to make the GET request.
+    url : str
+        The paginated endpoint URL.
+    params : Dict[str, Union[int, SortOrder]]
+        The query parameters for the GET request.
+
+    Returns
+    -------
+    A dictionary containing the following keys:
+
+        - "offset": int, representing starting point of returned items.
+        - "limit: int, representing max items to return."
+        - "items": list of int, representing timestamps.
+        - "total": int, representing total number of items.
+    """
```

Comment:
This docstring does not conform to the type of value returned by the method.

Reply:
Good catch!